### PR TITLE
Fix Locale Time Conversion error on PT-BR and Simplified Chineses

### PR DESCRIPTION
**Pull request by flavioislima**
This fixes an error when passing the wrong locale to Intl.DateTimeFormat() was breaking the game page after playing games.

-------------------------------------------------------------
Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [x] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [x] Created / Updated Tests (If necessary)
- [x] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by dragonDScript — APPROVED** [review]

File: src/screens/Game/TimeContainer/index.tsx [hunk 1]
```diff
@@ -21,11 +21,11 @@ type TimeStamp = {
...
 }
 
-function TimeContainer({game}: Props) {
-  const {t} = useTranslation('gamepage')
+function TimeContainer({ game }: Props) {
+  const { t } = useTranslation('gamepage')
   const hasPlayed = store.has(game)
 
-  if (!hasPlayed){
+  if (!hasPlayed) {
```
Comment (dragonDScript) [inline src/screens/Game/TimeContainer/index.tsx:28]: Can you use an arrow func here?

File: src/screens/Game/TimeContainer/index.tsx [hunk 2]
```diff
@@ -37,15 +37,16 @@ function TimeContainer({game}: Props) {
     timeStyle: 'short'
   };
   const firstPlayed = new Date(tsInfo.firstPlayed)
-  const firstDate = new Intl.DateTimeFormat(storage.getItem('language') || 'en', options).format(firstPlayed);
+  const language = storage.getItem('language')?.replace('_', '-').toLowerCase()
```
Comment (dragonDScript) [inline src/screens/Game/TimeContainer/index.tsx:40]: Is pt-br expected here? Shouldn't it be pt-BR?

**Review by dragonDScript — DISMISSED** [review]

File: src/screens/Game/TimeContainer/index.tsx (hunk 1 above)
Comment (flavioislima) [reply src/screens/Game/TimeContainer/index.tsx:28]: Why?

File: src/screens/Game/TimeContainer/index.tsx (hunk 2 above)
Comment (flavioislima) [reply src/screens/Game/TimeContainer/index.tsx:40]: Hmm, actually both work.

**Pull request merged**